assets/js/navbar.js

```diff
@@ -20,13 +20,22 @@ const controllerNavbar = (function (jQuery) {
       const dropdownToggle = dropdownMenu.find('.dropdown-toggle');
       const dropdownMenuContent = dropdownMenu.find('.dropdown-menu');
 
-      dropdownToggle.on('mouseenter', function () {
-        dropdownMenuContent.addClass('show');
-      })
-
-      dropdownMenu.on('mouseleave', function () {
-        dropdownMenuContent.removeClass('show');
-      })
+      // Toggle dropdown menu when clicking the dropdown toggle
+      dropdownToggle.on('click', function (e) {
+        e.preventDefault(); // Prevent default anchor behavior
+        
+        // Close all other dropdowns
+        jQuery('.dropdown-menu').not(dropdownMenuContent).removeClass('show');
+        
+        dropdownMenuContent.toggleClass('show');
+      });
+
+      // Hide dropdown menu when clicking outside the dropdown menu
+      jQuery(document).on('click', function (e) {
+        if (!dropdownMenu.is(e.target) && dropdownMenu.has(e.target).length === 0 && !dropdownToggle.is(e.target) && dropdownToggle.has(e.target).length === 0) {
```

Comment:
What do you think about make this code easier to follow, by extract the logic to variables with meaninfull name? 

For example: 
```
       const isDropdownMenuNotSelected = (!dropdownMenu.is(e.target) && dropdownMenu.has(e.target).length === 0 ; 
       const isNotdropdownToggle = !dropdownToggle.is(e.target) && dropdownToggle.has(e.target).length === 0;
       
        if isDropdownMenuNotSelected &&isNotdropdownToggle ) {

Reply:
Would be nice to create unit test for that, for fun and also to document the code, you can check the example we current have [here](https://github.com/WomenCodingCommunity/WomenCodingCommunity.github.io/blob/main/test/assets/code_of_conduct.test.js)